query/src/common/context_function.rs

```diff
@@ -46,6 +46,9 @@ impl ContextFunction {
             "current_user" => vec![Expression::create_literal(DataValue::String(
                 ctx.get_current_user()?.identity().to_string().into_bytes(),
             ))],
```

Comment:
New planner also need add the case.

Reply:
https://github.com/datafuselabs/databend/pull/5369/files#diff-37b754ad52b168f02e969eb2486df14bde8efb8c819a218053a1dcf79bff6b50R591

Reply:
ok